src/components/radio-button/radio-button-group/radio-button-group.ts

```diff
@@ -232,9 +232,11 @@ export class SbbRadioButtonGroup extends LitElement {
       return;
     }
 
-    this.value = initValue ?? this._radioButtons.find((radio) => radio.checked)?.value;
+    const radioButtons = this._radioButtons;
```

Comment:
Purpose of this variable? Performance impact should be negligible?

Reply:
Did expect to make you happy increasing performance 😃

Reply:
It‘s a getter which each time does a queryselectorall

Reply:
> Did expect to make you happy increasing performance 😃

More code vs better performance is always a trade-off 😄 
But this is fine then 👍